photo-app/app/assets/javascripts/credit_card_form.js

```diff
@@ -0,0 +1,58 @@
+let show_error, submitHandler;
+
+show_error = function (message) {
+  console.log(message);
+  if(!$("#flash-messages").length){
+    $('#main').prepend("<div id='flash-messages'></div>");
+  }
+
+  $("#flash-messages").html('<div class="alert alert-warning"><a class="close" data-dismiss="alert">×</a><div id="flash_alert">' + message + '</div></div>');
+  $('.alert').delay(5000).fadeOut(3000);
+
+  return false;
+};
+
+function stripeTokenHandler(token, form){
+  form.append($("<input type=\"hidden\" name=\"payment[token]\" />").val(token.id));
+  form.get(0).submit();
+};
+
+function createToken(form){
+  stripe.createToken(card).then(function(result) {
+    if(result.error){
+      show_error(result.error.message);
+      form.find("input[type=submit]").prop("disabled", false);
+    } else {
+      stripeTokenHandler(result.token, form);
+    }
+  });
+};
+
+submitHandler = function(event){
+  event.preventDefault();
+
+  var $form = $(event.target);
+  $form.find("input[type=submit]").prop("disabled", true);
+
+  //If Stripe was initialized correctly this will create a token using the credit card info
+  if(stripe){
+    createToken($form);
+  }else{
```

Comment:
```suggestion
  }else{
```
Why here no spacing but below have spacing? Keep it consistent. Choose one.